parsl/tests/test_bash_apps/test_stdout.py

```diff
@@ -67,6 +125,7 @@ def test_bad_stderr():
 
     if args.debug:
         parsl.set_stream_logger()
-
-    y = test_bad_stdout()
-    y = test_bad_stderr()
+    y = test_bad_stdout_specs()  # FIXME: Not sure how this should work when tested directly
```

Comment:
In general the __main__ tests don't always test everything - so if test_bad_stdout_specs() isn't going to work invoked this way, it is ok to leave it off the __main__ list.